### ISSUE TITLE
Game list ?

### ISSUE THREAD

**Issue by RobinLanglois** [isFakeAccount/psnawp#22]
Hello,
Is it possible to get the list of the games owned by the user ? (at least myself)
I couldn't find it.

Thank you, regards

**Comment by isFakeAccount** [comment]
I haven't added it yet. I can see if I can add it in next feature release.

**Comment by Lakrix** [comment]
Hey throphies for users would also be nice to have.

kind regard

**Comment by isFakeAccount** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add Games List](https://trello.com/c/qVU6N0ZE/2-add-games-list)

**Comment by Lakrix** [comment]
What is the api end point for fetching the players games?

**Comment by isFakeAccount** [comment]
@Lakrix 

```
https://m.np.playstation.com/api/gamelist/v2/users/<account_id>/titles?categories=ps4_game%2Cps5_native_game&fields=media&limit=20&offset=0&sort=-lastPlayedDate
```
I don't know all the details, like what categories there are and fields.

**Comment by isFakeAccount** [comment]
I haven't added this yet. Adding the trophy endpoint took more time than I thought. I will try to add this in version `1.2.0`

**Comment by isFakeAccount** [comment]
Fixed in version [1.2.0](https://github.com/isFakeAccount/psnawp/releases/tag/v1.2.0)
